<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SVM labeler &amp; trainer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="labelTab">
       <attribute name="title">
        <string>Label images</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="3">
         <widget class="QPushButton" name="newLabelsButton">
          <property name="text">
           <string>New labels</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="imagesLabeledLabel">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3" colspan="2">
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="loadImagesButton">
          <property name="text">
           <string>Load images</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0" colspan="2">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="4">
         <widget class="QPushButton" name="startLabelingButton">
          <property name="text">
           <string>Start labeling</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QPushButton" name="appendToLabelsButton">
          <property name="text">
           <string>Append to labels</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLabel" name="imageSizeLabel">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3" colspan="2">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="5" column="4">
         <widget class="QPushButton" name="bulkAddButton">
          <property name="text">
           <string>Bulk add</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" rowspan="6" colspan="2">
         <widget class="QLabel" name="imageLabel">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>200</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>9999999</width>
            <height>9999999</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="text">
           <string>Image will be shown here!</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="3" colspan="2">
         <widget class="QSpinBox" name="labelImageValueSpinbox">
          <property name="minimum">
           <number>-99</number>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Image:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Image size:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QPushButton" name="addLabelButton">
          <property name="text">
           <string>Add</string>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Image path:</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1" colspan="4">
         <widget class="QLabel" name="imagePathLabel">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="trainTab">
       <attribute name="title">
        <string>Train and test SVM</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="8" column="0" colspan="5">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="3" colspan="2">
         <widget class="QPushButton" name="loadTestDataButton">
          <property name="text">
           <string>Load test data</string>
          </property>
         </widget>
        </item>
        <item row="4" column="4">
         <widget class="QLabel" name="testResultsLabel">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3" colspan="2">
         <widget class="QPushButton" name="testSvmButton">
          <property name="text">
           <string>Test SVM</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Test results:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3" colspan="2">
         <widget class="QPushButton" name="loadSvmButton">
          <property name="text">
           <string>Load SVM</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QPushButton" name="trainSvmButton">
          <property name="text">
           <string>Train SVM</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QPushButton" name="loadTrainingDataButton">
          <property name="text">
           <string>Load training data</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" rowspan="8">
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
